k9mail/src/test/java/com/fsck/k9/notification/NotificationDataTest.java

```diff
@@ -267,6 +267,32 @@ public void testGetAllMessageReferences() throws Exception {
         assertEquals(messageReference8, messageReferences.get(8));
     }
 
+    @Test
+    public void testOverflowNotifications() {
```

Comment:
There's an `if` statement in `getSummaryOverflowMessagesCount()`. To properly test that code there should be at least two tests.